ingest/common/poolextractor/pool_extractor.go

```diff
@@ -55,7 +59,7 @@ func (p *poolExtractor) ExtractAll(ctx sdk.Context) (commondomain.BlockPools, er
 		CFMMPools:         cfmmPools,
 	}
 
-	return blockPools, nil
+	return blockPools, createdPoolIDs, nil
```

Comment:
Could we instead apply the creation metadata to `blockPools` here and return the pools that are already instrumented with the creation data?

This would reduce the API complexity while also making it more testable, making it possible to seamlessly cover this method with a unit test.